packages/evm/contracts/voting/CRVoting.sol

```diff
@@ -237,6 +373,21 @@ contract CRVoting is Controlled, ICRVoting {
         return _hashVote(_outcome, _salt);
     }
 
+    /**
+    * @dev Commit a vote on behalf of `_voter` for vote #`_voteId`
```

Comment:
Not sure if we do this in other places in these contracts, but would it be worth mentioning here that we assume the caller has authenticated the args already?